src/client/dfuse/ops/setxattr.c

```diff
@@ -42,7 +42,8 @@ dfuse_cb_setxattr(fuse_req_t req, struct dfuse_inode_entry *inode,
 		if (rc)
 			D_GOTO(err, rc);
 
-		if (dattr.da_type != DAOS_PROP_CO_LAYOUT_POSIX)
+		if (dattr.da_type != DAOS_PROP_CO_LAYOUT_POSIX &&
```

Comment:
It looks like duns_parse_attr() already does this check anyway so we could probably just remove this from here.